app-typescript/components/Label.tsx

```diff
@@ -3,19 +3,19 @@ import classNames from 'classnames';
 
 interface IProps {
     text?: string;
-    color?: string;  //  https://ui-framework.superdesk.org/#/components/colors
+    color?: string;
     size?: 'small' | 'normal' | 'large'; // defaults to 'normal'
     onClick?(): void;
-    noTransform?: boolean;
+    transform?: boolean;
```

Comment:
Can you restore the old name (`noTransform`)? The new one isn't accurate, because you would pass `transform={true}` and it wouldn't get transformed.